tendrl/ceph_integration/objects/pool/atoms/rename/__init__.py

```diff
@@ -0,0 +1,71 @@
+from tendrl.ceph_integration.manager.crud import Crud
+from tendrl.ceph_integration.manager.exceptions import \
+    RequestStateError
+from tendrl.commons import objects
+from tendrl.ceph_integration.objects.pool import Pool
+from tendrl.commons.event import Event
+from tendrl.commons.message import Message
+from tendrl.commons.objects import AtomExecutionFailedError
+
+
+class Rename(objects.BaseAtom):
+    obj = Pool
+    def __init__(self, *args, **kwargs):
+        super(Rename, self).__init__(*args, **kwargs)
+
+    def run(self):
+        pool_id = self.parameters['Pool.pool_id']
+        attrs = {}
+        attrs['name'] = self.parameters.get('Pool.poolname')
+        Event(
+            Message(
+                priority="info",
+                publisher=NS.publisher_id,
+                payload={
+                    "message": "Renaming the pool with pool-id %s."
```

Comment:
For old name we would need to fetch the pool from etcd. Anyway will add new name at least.